src/test/clojure/jarl/builtins/comparison_test.clj

```diff
@@ -0,0 +1,163 @@
+(ns jarl.builtins.comparison_test
+  (:require [clojure.test :refer [deftest is testing]]
+            [jarl.builtins.comparison :refer [builtin-equal builtin-neq builtin-lt builtin-lte builtin-gt
+                                              builtin-gte]]))
+
+(deftest builtin-equal-test
```

Comment:
Some additional negative tests would be nice.

Reply:
Added.